Azure-Samples/azure-ml-federated-learning
# To store aggregated models in a single folder

## Purpose

* Aggregated models are to be stored in a single folder on the datastore (path: 
 "< date>_<random_num>/round_<round_num>/model.pt")

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?


```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[x] Other... Please describe:
```

## How to Test
*  Get the code

```
git clone [repo-address]
cd [repo-name]
git checkout [branch-name]
npm install
```

* Test the code

Submit a pipeline and go to the storage account to see if the aggregated models are stored in a single folder.
